core/src/main/java/io/kestra/core/models/executions/Execution.java

```diff
@@ -91,6 +92,19 @@ public class Execution implements DeletedInterface, TenantInterface {
     @Builder.Default
     boolean deleted = false;
 
+    @Builder.Default
+    @With
+    @JsonInclude
+    Integer attemptNumber = 1;
+
+    @JsonSetter
+    public Integer getAttemptNumber() {
+        if (attemptNumber == null) {
+            return 1;
+        }
+        return attemptNumber;
+    }
```

Comment:
Why? The `@Builder.Default` should do the trick not?

Reply:
For execution that were created before the change ?